### PR TITLE
Setup transport

### PR DESCRIPTION
**Pull request by Korgath**
## Description

Implementing Transport functionality

## References

* #554 #561

### CODE REVIEW

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Transport/TransportTests.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+            // The wire
+            var bytes = channel.GetWrittenBytes();
+
+            // Receive
+            var receivingServiceCollection = new ServiceCollection();
+            receivingServiceCollection.ReceiveProtobuf<TestEnvelope>(
+               config => config
+                    .FromOneOf(envelope => envelope.TestMessagesCase)
+                   .WithParser(() => TestEnvelope.Parser));
```
Comment (cmgrd) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Transport/TransportTests.cs:51]: nit: misalignment

**Review by cmgrd — DISMISSED** [review]

**Review by kft — CHANGES_REQUESTED** [review]
Is it possible to keep transport/messaging/serialization concerns out of the application layer ? And move it to the infrastructure layer where it naturally belongs ?

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Transport/TransportTests.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Energinet.DataHub.MarketRoles.IntegrationTests.Transport
+{
+    public class TransportTests
+    {
+        [Fact]
+        public async Task Send_and_receive_must_result_in_same_transmitted_values()
+        {
```
Comment (kft) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Transport/TransportTests.cs:30]: Does it make sense to keep tests for infrastructure code ? This test makes perfect sence while implementing the code, but afterwards, it is just testing implementation details and not business logic/public behaviour. I will suggest deleting tests like these when implementation is done.

**Review by kft — APPROVED** [review]
LGTM! A comment regarding integration tests though.

**Review by erduna — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Transport/TransportTests.cs (hunk 2 above)
Comment (Korgath) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Transport/TransportTests.cs:30]: we'll have the discussion in the team

**Pull request merged**